students/steve_walker/lesson03/mailroom.py

```diff
@@ -55,6 +55,8 @@ def create_report():
     for record in donor_records:
         donor_summary.append([record[0], sum(record[1],0), 
             len(record[1]), sum(record[1],0)/len(record[1])])
+    donor_summary.sort(key = lambda donor_summary: donor_summary[1], 
+        reverse = True)
```

Comment:
👍 

couple other options you can use:
* define a sort function and pass for key= param
* use operator.itemgetter function


documentation:
https://docs.python.org/3/howto/sorting.html